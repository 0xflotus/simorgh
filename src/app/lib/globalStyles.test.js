import * as styledComponents from 'styled-components';

describe('injectGlobal', () => {
  it('should be called with args matching snapshot', async () => {
    jest.spyOn(styledComponents, 'injectGlobal');

    await import('./globalStyles');

    const injectGlobalArgs = styledComponents.injectGlobal.mock.calls[0];

    expect(injectGlobalArgs).toMatchSnapshot();
  });
});
