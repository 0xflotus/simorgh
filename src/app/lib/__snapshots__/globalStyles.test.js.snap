// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`globalStyles should call createGlobalStyle with args matching snapshot 1`] = `undefined`;
